aws-android-sdk-iot/src/main/java/com/amazonaws/mobileconnectors/iot/AWSIotMqttManager.java

```diff
@@ -1185,6 +1185,21 @@ public void resetReconnect() {
      */
     public void subscribeToTopic(String topic, AWSIotMqttQos qos,
             AWSIotMqttNewMessageCallback callback) {
+        subscribeToTopic(topic, qos, null, callback);
+    }
+
+    /**
+     * Subscribes to an MQTT topic
+     *
+     * @param topic The topic to which to subscribe.
+     * @param qos Quality of Service Level of the subscription.
+     * @param subscriptionStatusCallback Callback that will be notified when the subscribe has completed.
+     * @param callback Callback to be called when new message is received on this
+     *                 topic for this subscription.
```

Comment:
can you add a comment in the Javadoc which says when the callback is invoked and when the AmazonClientException is thrown?

Reply:
Done